client/src/pages/Campaigns.js

```diff
@@ -105,7 +136,7 @@ const Campaigns = () => {
       <Box className="tagsContainer" display="flex" justifyContent="center">
       </Box>
       <UserInputContainer className={classes.campaignList}>
-        <DataTable data={dataToRender}></DataTable>
+        <DataTable data={campaigns} func={test} type="campaign" campaignId={setSelectedId}></DataTable>
```

Comment:
It seems like the DataTable is getting away from the original idea of creating a generic component. Maybe instead of having logic in the data table to handle clicks based on type, we could pass in a function with each element in data that would get called when the row is clicked.